Classes/Emogrifier.php

```diff
@@ -911,13 +911,14 @@ private function attributeValueIsImportant($attributeValue)
      *
      * @return void
      */
-    private function copyCssWithMediaToStyleNode(\DOMDocument $xmlDocument, \DOMXPath $xPath, $css)
+    private function copyCssWithMediaToStyleNode(\DOMDocument $xmlDocument, \DOMXPath $xPath, $css, $imports = '')
```

Comment:
And why does the new parameter have a default value? Do you expects (at least) two different calls (one with the argument and one without it)?